config/serverless.es.yml

```diff
@@ -131,3 +131,6 @@ aiAssistantManagementSelection.preferredAIAssistantType: "observability"
 xpack.observabilityAiAssistantManagement.logSourcesEnabled: false
 xpack.observabilityAiAssistantManagement.spacesEnabled: false
 xpack.observabilityAiAssistantManagement.visibilityEnabled: false
+
+# Synonyms UI
+xpack.searchSynonyms.enabled: true
```

Comment:
Mind that we're also adding it the Serverless Search distribution that will be released next week. Is that intended?

Happy to approve if so.

Reply:
Yes it is. It was disabled in case if this was not finished in time. It will still be behind a kibana UI setting also

Reply:
This is not need with the [change](https://github.com/elastic/kibana/pull/208564/files#diff-f4d81bbd434c6caf7c03d7e3776bb808b9dcea06113d63718ab51b9b4581b7c8R12) in the flyout PR. but you may need to explicitly disable this plugin for the other solutions since you are enabling it by default. 

@afharo will this plugin be disabled through the new distribution process? what about until then?